com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/RenderPass/DrawRenderersCustomPass.cs

```diff
@@ -66,6 +66,46 @@ public enum ShaderPass
         /// </summary>
         public bool depthWrite = true;
 
+        /// <summary>
+        /// Override the stencil state of the objects.
+        /// </summary>
+        public bool overrideStencil = false;
+
+        /// <summary>
+        /// Stencil reference value. Be careful when using this value to write in the stencil buffer to not overwrite HDRP stencil bits.
+        /// </summary>
+        public int stencilReferenceValue = (int)UserStencilUsage.UserBit0;
+
+        /// <summary>
+        /// Write mask of the stencil.
+        /// </summary>
+        public int stencilWriteMask = (int)(UserStencilUsage.UserBit0 | UserStencilUsage.UserBit1);
+
+        /// <summary>
+        /// Read mask of the stencil
+        /// </summary>
+        public int stencilReadMask = (int)(UserStencilUsage.UserBit0 | UserStencilUsage.UserBit1);
```

Comment:
same remark than before about creating a reservert userBitMask. and IIRC there is one currently